src/v/raft/tests/append_entries_test.cc

```diff
@@ -831,7 +831,9 @@ struct request_ordering_test_fixture : public raft_test_fixture {
     }
 
     ss::future<> replicate_indexed_batches(
-      consensus_ptr c, int count, raft::consistency_level consistency) {
+      consensus_ptr c,
+      [[maybe_unused]] int count,
```

Comment:
all the callers pass in `20` here - can we just use the parameter instead of the hardcoded value?